style.css

```diff
@@ -0,0 +1,619 @@
+}
+
+.link a:hover {
+  color: #172b4d;
+}
+
+.astaricks {
+  max-width: 100%;
+}
+
+.contruct {
+  margin: 25px 0;
+  display: grid;
+  row-gap: 25px;
+  padding: 10px;
+  background: #e5e5e5;
+  justify-items: center;
+}
+
+.topic {
+  background: #fff;
+  border-radius: 8px;
+  min-height: 5em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+}
+
+.topic2 {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+}
+
+.topic3 {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+}
+
+.topic4 {
+  background: #fff;
+  border-radius: 10px;
+  min-height: 3em;
+  border: 1px solid #c6c6c6;
+  padding: 20px 15px;
+  display: grid;
+}
+
+.summary {
+  grid-column: 1/-1;
+  justify-self: center;
+  align-self: center;
+}
+
+.summary img {
+  max-width: 100%;
+}
+
+.instruction {
+  grid-column: 1/-1;
+  align-self: center;
+  padding: 10px;
+}
+
+.employed {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  margin: 10px 0;
+  color: #172b4d;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 80px;
+}
+
+.mission {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  display: flex;
+  overflow: hidden;
+  color: #7a869a;
+}
+
+.body {
+  margin: 10px 0;
+  padding: 10px 0;
+  color: #344563;
+  font-family: Poppins, sans-serif;
+}
+
+.code_use {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 10px 0;
+}
+
+.code_use > li {
+  padding: 8px;
+  list-style: none;
+  color: #6070ff;
+  border-radius: 8px;
+  margin-right: 8px;
+  background: #ebebff;
+  font-size: 14px;
+  line-height: 16px;
+}
+
+.tapbutton {
+  cursor: pointer;
+  font-size: 18px;
+  margin-top: 20px;
+  color: #396df2;
+  padding: 10px 20px;
+  background: #fff;
+  border-radius: 10px;
+  border: 1px solid #396df2;
+}
+
+.tapbutton1 {
+  cursor: pointer;
+  font-size: 18px;
+  margin-top: 20px;
+  color: #fff;
+  padding: 10px 20px;
+  background: #4053fc;
+  border-radius: 8px;
+  border: 1px solid #396df2;
+}
+
+a:disabled:hover {
+  background: #fff;
+  color: #5e6c84;
+}
+
+.information {
+  display: grid;
+  row-gap: 5px;
+  padding: 14px;
+}
+
+.boldtext {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-size: 18px;
+  line-height: 20px;
+  font-weight: bold;
+  margin: 10px 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+.arrowkey {
+  width: 10px;
+  height: 10px;
+  overflow: hidden;
+}
+
+.duck {
+  width: 10px;
+  height: 14px;
+  overflow: hidden;
+}
+
+#container {
+  display: flex;
+  color: #fff;
+  padding: 20px;
+  margin-top: 20px;
+  background-size: contain;
+  border-top-left-radius: 70px;
+  background-repeat: no-repeat;
+  background-position: bottom -1px right -13px;
+  background-image: url("./images/contact\ form\ background\ shapes.svg");
+  background-color: #6070ff;
+  border: 24px;
+}

```

Comment:
In the HTML file, you are using a class named `container` but in the CSS file, you are targeting it as it is an id.
- [ ] Please correct this aspect, deciding what you want to use: a class or an id.

- [ ] Also, please make sure to use the whole width of the screen for the contact form in the desktop view:

Your implementation | Figma Design
:------:|:------:
![image](https://user-images.githubusercontent.com/83596135/203150993-f2456a82-5dd8-4c89-93cf-e2becb79f4e2.png) | ![image](https://user-images.githubusercontent.com/83596135/203151052-e7f8cd04-9897-4b79-a948-37a6d4c0ad42.png)